server.go

```diff
@@ -5,12 +5,14 @@ package main
 
 import (
 	"context"
-	"github.com/camirmas/go_stop/models"
-	"github.com/graphql-go/graphql"
-	"github.com/graphql-go/handler"
+	"fmt"
```

Comment:
I didn't mean to move these imports around, I think `gofmt` or `goimports` decided that they should get moved around 🤷‍♂️